src/Core/TileMesh.js

```diff
@@ -29,7 +29,7 @@ function TileMesh(geometry, params) {
 
     this.obb = this.geometry.OBB.clone();
 
-    this.boundingSphere = this.OBB().box3D.getBoundingSphere();
+    this.OBB().box3D.getBoundingSphere(this.boundingSphere);
```

Comment:
The following should be used imo:

```js
this.boundingSphere = this.OBB().box3D.getBoundingSphere(new THREE.Sphere());
```

Reply:
I realize that ` this.OBB().box3D.getBoundingSphere(this.boundingSphere)` is not even correct (as `this.boundingSphere` is `undefined`). No?

Reply:
Yep. That's why travis is red currently.

Reply:
@mbredif you use THREE v91?

Reply:
> you use THREE v91?

Yes, by mistake :)